Log the status of the import process.

juangallostra opened the pull request:
## Description of the issue/feature this PR addresses

When importing data from another instance the user cannot know how many objects have been imported out from the total amount to be imported.

## Current behavior before PR

No info is logged that informs the user on which is the the current status of the import process (How many objects out of the total number of objects have been imported).

## Desired behavior after PR is merged

The current status of the import process is logged into the terminal. This is done by showing the number of objects already imported out of the total number of objects to be imported.

--
I confirm I have tested the PR thoroughly and coded it according to [PEP8][1]
standards.

[1]: https://www.python.org/dev/peps/pep-0008

Commit:
Log the status of the import process.

This is done by showing the number of objects already imported out of the total number of objects to be imported.

ramonski commented on src/senaite/sync/importstep.py:
```diff
@@ -113,8 +113,8 @@ def _import_data(self):
         ordered_uids = storage["ordered_uids"]
-
-        for r_uid in ordered_uids:
+        
```
Please remove trailing whitespaces

Commit:
Add blank line

ramonski commented on src/senaite/sync/importstep.py:
```diff
@@ -133,7 +133,7 @@ def _import_data(self):
                             self._non_commited_objects, len(ordered_uids)))
                 self._non_commited_objects = 0
-
+            logger.info("Imported: {} / {}".format(item_count+1, len(ordered_uids)))
```
You are calculating here the length for each item in the count. You can speed up things by defining a variable outside of the loop.

Commit:
Store total object count in a variable

This way we avoid having to compute it by checking the uid list lenght every time we need it.

Commit:
Remove trailing whitespace
